docs/content/docs/install/configuration/feed_configuration.md

```diff
@@ -43,6 +43,11 @@ feeds:
   read_timeout_seconds: 60
 ```
 
+***Note:*** Engine now has a new feed for Grype. This feed will only be synced if the
```

Comment:
the old style config was moved to a new location. need to update the examples here to reflect that location. `services.policy_engine.vulnerabilities.sync.data`